### ISSUE TITLE
I trained 1 class with yolov3, the dataset contains 2500 images

### ISSUE THREAD

**Issue by ChenCong7375** [AlexeyAB/darknet#1630]
Q1: I trained the model for 40000 batches, but it seemed that it had overfitted. How many batches should I set?
Q2: The final loss was about 0.19,What should I do to minimize the loss value?
